src/pages-and-resources/discussions/LtiConfigForm.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage, injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import {
+  Form, Hyperlink,
+} from '@edx/paragon';
+import { useFormik } from 'formik';
+import * as Yup from 'yup';
```

Comment:
I don't have any suggestions (and I've already looked up the docs), but this is an unfortunately opaque library name.

No ask here, just shouting into the void ;)

Reply:
Yeah, while cute, it's totally uninformative.